resources/qml/dialogs/UserProfile.qml

```diff
@@ -292,13 +300,13 @@ ApplicationWindow {
             Image {
                 Layout.preferredHeight: 16
                 Layout.preferredWidth: 16
-                source: ((model.verificationStatus == VerificationStatus.VERIFIED) ? "image://colorimage/:/icons/icons/ui/lock.png?green" : ((model.verificationStatus == VerificationStatus.UNVERIFIED) ? "image://colorimage/:/icons/icons/ui/unlock.png?yellow" : "image://colorimage/:/icons/icons/ui/unlock.png?red"))
+                source: ((model.verificationStatus == VerificationStatus.VERIFIED) ? "image://colorimage/:/icons/icons/ui/lock.png?green" : ((model.verificationStatus == VerificationStatus.UNVERIFIED) ? "image://colorimage/:/icons/icons/ui/unlock.png?yellow" : ((model.verificationStatus == VerificationStatus.SELF) ? "image://colorimage/:/icons/icons/ui/checkmark.png?green" : "image://colorimage/:/icons/icons/ui/unlock.png?red")))
```

Comment:
How about... we convert that to an if else block, with returns for the respective icons?